client/components/Coverages/CoverageArrayInput.tsx

```diff
@@ -217,22 +217,23 @@ class CoverageArrayInputComponent extends React.Component<IProps, IState> {
                 addButtonComponent={CoverageAddButton}
                 onRemoveAssignment={this.props.onRemoveAssignment}
                 addButtonProps={{
-                    contentTypes,
-                    defaultDesk,
-                    onPopupOpen,
-                    onPopupClose,
-                    preferredCoverageDesks,
-                    newsCoverageStatus,
-                    field,
-                    value,
-                    onChange,
-                    createCoverage,
-                    desks,
-                    users,
-                    coverageAddAdvancedMode,
-                    setCoverageAddAdvancedMode,
-                    language,
-                    editorType,
+                    contentTypes: contentTypes,
+                    defaultDesk: defaultDesk,
+                    onPopupOpen: onPopupOpen,
+                    onPopupClose: onPopupClose,
+                    preferredCoverageDesks: preferredCoverageDesks,
+                    newsCoverageStatus: newsCoverageStatus,
+                    field: field,
+                    value: value,
+                    onChange: onChange,
+                    createCoverage: createCoverage,
+                    desks: desks,
+                    users: users,
+                    coverageAddAdvancedMode: coverageAddAdvancedMode,
+                    setCoverageAddAdvancedMode: setCoverageAddAdvancedMode,
+                    language: language,
+                    editorType: editorType,
+                    eventLanguages: event?.languages ?? [],
```

Comment:
this is kinda redundant, I think it was cleaner as it was before. 
```suggestion
                    contentTypes: contentTypes,
                    defaultDesk: defaultDesk,
                    onPopupOpen: onPopupOpen,
                    onPopupClose: onPopupClose,
                    preferredCoverageDesks: preferredCoverageDesks,
                    newsCoverageStatus: newsCoverageStatus,
                    field: field,
                    value: value,
                    onChange: onChange,
                    createCoverage: createCoverage,
                    desks: desks,
                    users: users,
                    coverageAddAdvancedMode: coverageAddAdvancedMode,
                    setCoverageAddAdvancedMode: setCoverageAddAdvancedMode,
                    language: language,
                    editorType: editorType,
                    eventLanguages: event?.languages ?? [],
```

Reply:
I have it like this because of `eventLanguages: event?.languages ?? [],`, we can't have mixed properties where some you just pass and then others you specify like I've done fro `eventLanguages`, thus I have to keep them like this